src/scp/BallotProtocol.cpp

```diff
@@ -2036,7 +2036,10 @@ BallotProtocol::getJsonQuorumInfo(NodeID const& id, bool summary, bool fullKeys)
         {
             if (!summary)
             {
-                missing.append(mSlot.getSCPDriver().toShortString(n));
+                std::string nodeID =
```

Comment:
maybe it would be cleaner to just make it that the conversion function takes a bool:
```
nodeID = mSlot.getSCPDriver().toStrKey(n, fullKeys)
```

that way we avoid having the checks all over the place

Reply:
done